docs/overview/architecture.md

```diff
@@ -111,7 +111,37 @@ When the user searches a vector from Vald:
 15. Vald Egress Filter will return the filtered result to the Vald Filter Gateway.
 16. Vald Filter Gateway will return the final result to the Vald Ingress.
 
-<!-- ### Update -->
+### Update
+
+<img src="../../assets/docs/update_flow.png" />
+
+When the user updates a vector from Vald:
+
+1. Vald Ingress receives the request from the user. The request includes the new vector and the vector ID to be updated.
+2. Vald Ingress will forward the request to the Vald Filter Gateway to pre-process the request data.
+3. Vald Filter Gateway will forward the request to the user-defined Vald Ingress Filter. After the Vald Ingress Filter received the request, it will perform the pre-processing logic defined by the user, for example, padding the vector to match the vector dimension in Vald.
+4. After the request is processed by the user-defined Vald Ingress Filter, the result will return to the Vald Filter Gateway.
+5. Vald Filter Gateway will forward the processed data to the Vald Meta Gateway. Vald Meta Gateway is used to resolve the internal used UUID from Vald Meta to the user inserted vector ID in the Insert Step.
+6. Vald Meta Gateway will forward the request to the Vald Meta to confirm whether the metadata, which contains the request ID(s), exists or not.
+7. Vald Meta gets the UUID by the request ID. It returns an error if no UUID is found.
+8. If Vald Meta Gateway gets the UUID(s), Vald Meta Gateway will forward the request with the UUID(s) to the Vald Backup Gateway.
```

Comment:
should we change all `UUID(s)` to `UUID` if it can only process one vector (one UUID) at the same request?

Reply:
yes, but I think we should rewrite another PR.
because this PR is only update data flow document:bow:

cc: @vankichi

Reply:
I mean in update flow only, I think we have `bulkInsert` but I am not sure do we have `bulkUpdate`

Reply:
ok. 
In this case, I think the UUID is correct, so I will correct it. What do you think?

Reply:
@vankichi what do you think?

Reply:
@kevindiu @hlts2 
I thinks using UUID(s) is correct because of https://github.com/vdaas/vald/blob/feature/architecture/divide-gateway-and-refactor/pkg/gateway/meta/handler/grpc/handler.go#L343